packages/react/src/components/Table/TablePropTypes.js

```diff
@@ -49,15 +49,15 @@ export const RowActionsStatePropTypes = PropTypes.arrayOf(
 
 export const EmptyStatePropTypes = PropTypes.oneOfType([
   PropTypes.shape({
-    message: PropTypes.node.isRequired,
-    messageBody: PropTypes.node,
+    message: PropTypes.oneOfType([PropTypes.node, PropTypes.string]).isRequired,
```

Comment:
This should be only string.
message: PropTypes.string.isRequired,

Reply:
Since this is a minor version bump we must keep  the node proptype for backwards compatibility.

Reply:
@jessieyan Thank you for commenting on this. I changed `PropTypes.node` to `PropTypes.oneOfType([PropTypes.node, PropTypes.string])` for backward compatibility. In the last commits I modified `EmptyState` component itself, for prop types of `string` and `node`.